src/app/components/elements/inputs/EditablePageTitle.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect, useState } from "react";
+import { Button, Input, Row } from "reactstrap";
+
+export const EditablePageTitle = ({
+  currentPageTitle,
+  onEdit,
+}: {
+  currentPageTitle: string;
+  onEdit: (arg0: string) => void;
+}) => {
+  const [newTitle, setNewTitle] = useState<string>(currentPageTitle);
+  const [toggleEdit, setToggleEdit] = useState<boolean>(false);
+
+  useEffect(() => {
+    setNewTitle(currentPageTitle ?? "");
```

Comment:
I don't fully understand this one. If `currentPageTitle` cannot be undefined or null, how come we need the nullish coalescing operator?

Reply:
Yep, that was leftover from something else I was trying. Fixed!